hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/MiniJournalCluster.java

```diff
@@ -64,7 +65,17 @@ public static class Builder {
     public Builder(Configuration conf) {
       this.conf = conf;
     }
-    
+
+    public Builder(Configuration conf, TemporaryFolder baseDir) {
```

Comment:
i don't want to add junit dependencies here. better to take a File ref and pass it in when used

Reply:
I can remove the JUnit dependency, but I was thinking it would be useful to differentiate between those instances that had been updated to use the TemporaryFolder handling.

Reply:
no idea where it is used, but in tests in other applications, they may not use junit or may be on a different version of it

Reply:
I've stripped all the JUnit dependencies, including one instance that used a JUnit assertion.